lib/hash.js

```diff
@@ -1,8 +1,24 @@
 'use strict'
 
+function getInvalidatingOptions (config) {
+  return [
```

Comment:
I think this list needs to be expanded to all configs options used in `NYC.prototype._createInstrumenter` as they all effect the output.